qiskit/synthesis/evolution/qdrift.py

```diff
@@ -68,6 +68,7 @@ def __init__(
         ) = None,
         seed: int | None = None,
         wrap: bool = False,
+        preserve_order: bool = True,
```

Comment:
This one defaults to `True` that's just inconsistent, but I think the expected value.